package.json

```diff
@@ -19,10 +19,20 @@
     "node": ">=20",
     "pnpm": ">=9"
   },
+  "packageManager": "pnpm@10.12.4",
   "pnpm": {
     "patchedDependencies": {
       "react-plotly.js": "patches/react-plotly.js.patch"
+    },
+    "overrides": {
+      "react": "^19.1.0",
+      "react-dom": "^19.1.0",
+      "vite": "npm:rolldown-vite@latest"
+    },
```

Comment:
Ah, I see the overrides here. I think this will also work, would just be nice to have the imports in the vite configs still from `vite`.

Reply:
yep, moved to that